src/systems/bit-media-frames.js

```diff
@@ -229,10 +316,16 @@ function mediaTypesOf(world, entities) {
   return mask;
 }
 
+export function cleanupMediaFrame(obj) {
+  obj.traverse(child => {
+    disposeNode(child);
+  });
+}
+
 const takeOwnershipOnTimeout = new Map();
 const heldQuery = defineQuery([Held]);
 // const droppedQuery = exitQuery(heldQuery);
-export function mediaFramesSystem(world) {
+export function mediaFramesSystem(world, physicsSystem) {
```

Comment:
Not a blocking issue, but a comment for future clean up.

I think ideally a system shouldn't have a dependency with another system.